Tasks/AzureAppServiceSettingsV1/README.md

```diff
@@ -0,0 +1,104 @@
+#  Azure App Service Settings
+
+## Overview
+
+The Azure App Service Settings task is used to update different Azure App Service settings for [Web Apps](https://azure.microsoft.com/en-in/documentation/articles/app-service-web-overview/). The task works on cross platform Azure Pipelines agents running Windows, Linux or Mac.
+
+The task works for [ASP.NET](https://www.visualstudio.com/en-us/docs/release/examples/azure/azure-web-apps-from-build-and-release-hubs), [ASP.NET Core](https://www.visualstudio.com/en-us/docs/release/examples/azure/aspnet-core10-azure-web-apps), PHP, Java, Python, Go and [Node.js](https://www.visualstudio.com/en-us/docs/release/examples/nodejs/node-to-azure-webapps) based web applications.
```

Comment:
Since package is not an input. You may remove this. Also get this reviewed by @N-Usha

Reply:
Got it reviewed. Made the changes.